seqr/management/tests/load_rna_seq_tpm_tests.py

```diff
@@ -23,41 +23,43 @@ def test_command(self, mock_open, mock_logger, mock_utils_logger, mock_gzip_open
-            'GTEX-001\t\tENSG00000240361\t3.1\tinvalid\n',
-            'NA19675_D2\t\tENSG00000233750\t1.04\tmuscle\n',
-            'NA19677\t\tENSG00000233750\t5.31\tmuscle\n',
-            'GTEX-001\t\tENSG00000233750\t7.8\tmuscle\n',
+            'NA19675_D2\t1kg project nåme with uniçøde\t\tENSG00000240361\t12.6\t\n',
+            'NA19678_D1\t1kg project nåme with uniçøde\t\tENSG00000233750\t 6.04\twhole_blood\n',
+            'NA19678_D1\t1kg project nåme with uniçøde\t\tENSG00000240361\t0.0\twhole_blood\n',
+            'NA19677\t1kg project nåme with uniçøde\t\tENSG00000240361\t0.0\tinvalid\n',
+            'GTEX-001\t1kg project nåme with uniçøde\t\tENSG00000240361\t3.1\tinvalid\n',
+            'NA19675_D2\t1kg project nåme with uniçøde\t\tENSG00000233750\t1.04\tmuscle\n',
+            'NA19677\t1kg project nåme with uniçøde\t\tENSG00000233750\t5.31\tmuscle\n',
+            'GTEX-001\t1kg project nåme with uniçøde\t\tENSG00000233750\t7.8\tmuscle\n',
+            'NA19678_D1\tTest Reprocessed Project\t\tENSG00000240361\t0.2\twhole_blood\n',
         ]
 
         with self.assertRaises(ValueError) as e:
             call_command('load_rna_seq_tpm', RNA_FILE_ID)
-        self.assertEqual(str(e.exception), 'Invalid file: missing column(s) TPM, gene_id, sample_id, tissue')
+        self.assertEqual(str(e.exception), 'Invalid file: missing column(s) TPM, gene_id, project, sample_id, tissue')
 
-        mock_gzip_file.__iter__.return_value[0] = 'sample_id\tindividual_id\tgene_id\tTPM\ttissue\n'
+        mock_gzip_file.__iter__.return_value[0] = 'sample_id\tproject\tindividual_id\tgene_id\tTPM\ttissue\n'
         with self.assertRaises(ValueError) as e:
             call_command('load_rna_seq_tpm', RNA_FILE_ID)
         self.assertEqual(str(e.exception), 'Sample NA19675_D2 has no tissue type')
 
-        mock_gzip_file.__iter__.return_value[1] = 'NA19675_D2\t\tENSG00000240361\t12.6\tfibroblasts\n'
+        mock_gzip_file.__iter__.return_value[1] = 'NA19675_D2\tproject\t\tENSG00000240361\t12.6\tfibroblasts\n'
         with self.assertRaises(ValueError) as e:
             call_command('load_rna_seq_tpm', RNA_FILE_ID)
         self.assertEqual(str(e.exception), 'Mismatched tissue types for sample NA19675_D2: fibroblasts, muscle')
 
-        mock_gzip_file.__iter__.return_value[1] = 'NA19675_D2\t\tENSG00000240361\t12.6\tmuscle\n'
+        mock_gzip_file.__iter__.return_value[1] = 'NA19675_D2\t1kg project nåme with uniçøde\t\tENSG00000240361\t12.6\tmuscle\n'
         with self.assertRaises(ValueError) as e:
             call_command('load_rna_seq_tpm', RNA_FILE_ID)
         self.assertEqual(str(e.exception), 'Unable to find matches for the following samples: NA19677, NA19678_D1')
 
-        mock_open.return_value.__enter__.return_value.__iter__.return_value = ['NA19678_D1\tNA19678']
+        mock_open.return_value.__enter__.return_value.__iter__.return_value = ['NA19678_D1\tNA19678', 'NA19677\tNA20877']
```

Comment:
how come you are adding a mapping for the new fixture data instead of just adding fixture data that works without mapping?

Reply:
The unmatched sample will be found in the previous test without the mapping. I can merge these two tests.